cfgov/v1/forms.py

```diff
@@ -126,7 +126,7 @@ def __init__(self, *args, **kwargs):
         self.hostname = kwargs.pop('hostname')
         self.base_query = kwargs.pop('base_query')
         super(FilterableListForm, self).__init__(*args, **kwargs)
-        page_ids = CFGOVPage.objects.live_shared(self.hostname).values_list('id', flat=True)
+        page_ids = self.base_query.live_shared(self.hostname).values_list('id', flat=True)
```

Comment:
Should this be using the `self.base_query()` method instead of the `self.base_query` member? Why do both of these exist?